kyo-core/shared/src/main/scala/kyo/Adder.scala

```diff
@@ -4,7 +4,7 @@ import java.util.concurrent.atomic as j
 
 /** A wrapper for Java's LongAdder.
   */
-final case class LongAdder private (unsafe: LongAdder.Unsafe) extends AnyVal:
```

Comment:
I've tried `opaque type LongAdder = LongAdder.Unsafe` but had trouble with compilation issues with `inline`. It's not ideal that we'll have these wrapper allocations but we can follow up later in case we see them as relevant in profiling sessions.